deep_qa/run.py

```diff
@@ -150,13 +151,17 @@ def score_dataset(param_path: str, test: str=False, model_class=None):
 
     Returns
     -------
-    Numpy arrays of model predictions in the format of model.outputs.
-
+    predictions: numpy.array
+        Numpy array of model predictions in the format of model.outputs (typically one array, but
+        could be List[numpy.array] if your model has multiple outputs).
+    dataset: Dataset
```

Comment:
Because of how sphinx / numpydoc processes parameters, you actually don't put double backticks around parameter types.  See http://deep-qa.readthedocs.io/en/latest/run.html.